schemas/eholdings/e-resources/resourcesRelationships.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Metadata of relationships for a given object",
+  "description": "Metadata of relationships for a given object",
+  "ResourcesRelationships": {
+    "type": "object",
```

Comment:
I'm not sure that this schema will be used for export purposes. I propose to check if it is possible to exclude not used properties from kb-ebsco schemas

Reply:
I think this property using for AccesType field
Here endpoint documentation: [Resources by id](https://s3.amazonaws.com/foliodocs/api/mod-kb-ebsco-java/resources.html#eholdings_resources__resourceid__get)